routes/doc.tsx

```diff
@@ -42,7 +42,7 @@ type ImgRoutes =
   | "/img/:proto(deno)/:host/~/:item+";
 
 const RE_STD = /^std(?:@?([^/]+))?/;
-const RE_X_PKG = /^x\/([a-zA-Z_]{3,})(?:@?([^/]+))?/;
+const RE_X_PKG = /^x\/([^@\/]{3,})(?:@?([^/]+))?/;
```

Comment:
I feel `[^@\/]` is too permissive as it allows the name like `%%%`. How about `[a-zA-Z0-9_]`?

Reply:
Sorry for the late reply! I was actually thinking the same thing since I wrote the original (I based the pattern on the URLPattern routes). 

What about `[a-zA-Z0-9-_.]`? I'm not sure what pattern is used for module submissions (other than a 3 char minimum), but ideally I think it should be the same as that.